src/hooks/useENS.ts

```diff
@@ -0,0 +1,25 @@
+import { ethers } from 'ethers';
+import { useEffect, useState } from 'react';
+import { EthNetworks } from 'services/web3/types';
+import { buildAlchemyUrl } from 'services/web3/wallet/connectors';
+
+const useENS = (address: string) => {
+  const provider = new ethers.providers.JsonRpcProvider(
+    buildAlchemyUrl(EthNetworks.Mainnet)
```

Comment:
@RanCohenn any comments on this regarding websockets? Can we utilise the `getProvider` helper here?